e2e/utils/TestUtils.ts

```diff
@@ -61,3 +61,12 @@ export async function tool(tool: string, ...args: string[]): Promise<string> {
 export async function kubectl(...args: string[] ): Promise<string> {
   return await tool('kubectl', ...args);
 }
+
+/**
+ * Run `helm` with given arguments.
+ * @returns standard output of the command.
+ * @example await helm('version')
+ */
+export async function helm(...args: string[] ): Promise<string> {
```

Comment:
Consider making a function that returns a tool; something like

```ts
function makeTool(tool: string): (...args: string[]) => Promise<string> {
  return tool.bind(null, tool);
}
export const kubectl = makeTool('kubectl');
export const helm = makeTool('helm');
```

(This may be overkill though.)